ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -4,6 +4,15 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# TODO remove unnecessary variables
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export AWS_REGION=us-east-1
```

Comment:
We don't need this; the installer should get the AWS region out of the `metadata.json` file (that `create cluster` spits out, and which is hopefully getting passed through via `${SHARED_DIR}`).